src/spatial/explog.hpp

```diff
@@ -206,7 +207,7 @@ namespace se3
     Matrix3Like & Jout = const_cast<Matrix3Like &>(Jlog.derived());
     typedef typename Matrix3Like::Scalar Scalar3;
     
-    if (theta < 1e-6)
+    if (theta < TaylorSeriesExpansion<Scalar>::template precision<4>())
       Jout.setIdentity();
```

Comment:
Here, shouldn't it be ?
```cpp
Jout.diagonal().setConstant(2 - theta*theta / Scalar(6));
Jout(0,1) = -log(2); Jout(1,0) =  log(2);
Jout(0,2) =  log(1); Jout(2,0) = -log(1);
Jout(1,2) = -log(0); Jout(2,1) =  log(0);
Jout /= Scalar3(2);
double alpha = Scalar(1)/Scalar(12) + theta*theta / Scalar(720);
Jout += alpha * log * log.transpose();
```

Reply:
Yes, I agree. I don't know why is there. It seems you did that (git blame).
Anyway, I will add this line to my PR.